### PR TITLE
Add documentation on rise-presentation-play/stop events

### PR DESCRIPTION
**Pull request by andrecardoso**

### CODE REVIEW

**Comment by andrecardoso** [comment]
[(1) Create PUD documentation for designers](https://trello.com/c/rNofaTMw/173-1-create-pud-documentation-for-designers)

File: README.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ RisePlayerConfiguration sends the following events:
 
 - **rise-components-ready**: Sent once to the window object when Local Messaging connection has been accomplished for the first time, and instructs all components in the page that they can begin doing requests like watch, licensing and logging. Preview mode also sends this event even if there is no Local Messaging connection, so components can start working in this environment.
 - **rise-local-messaging-connection**: Sent to the window object when Local Messaging connection status has changed. It provides an argument with a nested property *detail.isConnected* that indicates if connection to Local Messaging was successful or not. This event may be sent multiple times depending on the status of the connection.
+- **rise-presentation-play**: Sent to the window object when running on Viewer and presentation starts. It is useful to start animations, timers and reset the state of the template.
+- **rise-presentation-stop**: Sent to the window object when running on Viewer and presentation stops. It determines that the template is no longer showing and can be used to stop timers, animations or other behavior that is not desired when the presentation is not playing.
```
Comment (olegrise) [inline README.md]: @andrecardoso same here "...and each HTML component"

File: README.md [hunk 2]
```diff
@@ -49,6 +49,8 @@ RisePlayerConfiguration sends the following events:
 
 - **rise-components-ready**: Sent once to the window object when Local Messaging connection has been accomplished for the first time, and instructs all components in the page that they can begin doing requests like watch, licensing and logging. Preview mode also sends this event even if there is no Local Messaging connection, so components can start working in this environment.
 - **rise-local-messaging-connection**: Sent to the window object when Local Messaging connection status has changed. It provides an argument with a nested property *detail.isConnected* that indicates if connection to Local Messaging was successful or not. This event may be sent multiple times depending on the status of the connection.
+- **rise-presentation-play**: Sent to the window object when running on Viewer and presentation starts. It is useful to start animations, timers and reset the state of the template.
```
Comment (olegrise) [inline README.md]: @andrecardoso should it say "Sent to the window object **and each HTML component**"?

https://github.com/Rise-Vision/common-template/blob/ce147e0fa38c7224cc60f3e75e2b98d066e5f2fb/src/rise-viewer.js#L24

**Comment by andrecardoso** [comment]
@olegrise thanks! please re-review

**Pull request merged**